samples/Basic.Tests/TestHost.cs

```diff
@@ -22,8 +22,8 @@ private IServiceProvider CreateProvider()
 		{
 			var services = new ServiceCollection();
 			services.AddLogging();
-			services
-				.AddIdentity<AppUser, AppRole>()
+			services.AddIdentityCore<AppUser>(_ => { })
+				.AddRoles<AppRole>()
 				.AddUserStore<InMemoryUserStore<AppUser, AppRole>>()
```

Comment:
> AddIdentityCore will probably be used for apps that only provide REST API's

That's what I needed to know. Haven't yet had the time to go deep into how things changed in 2.0 😅 

Agreed. Another sample for usage with Core would be better, but it's also not that much needed. So feel free to do whatever is appropriate.